README.md

```diff
@@ -210,3 +211,84 @@ To create a valid order we should use appropriate builders, and specify our data
     const response = await orders.create(request);
 
 As one can see, splitting up the builders over multiple frontend components/views is easy (and encouraged!) to do.
+
+### Authenticating using Swedish BankId
+
+Bank id authentication is performed using three methods: `auth`, `collect` and `cancel`. In short, `auth` starts a authentication process, `collect` retrieves status information about the process and `cancel` aborts a process. [More information about this can be found here.](https://www.bankid.com/bankid-i-dina-tjanster/rp-info).
+
+#### Starting a BankId authentication process
+
+```
+    const request = bankid
+        .newAuthRequest()
+        .withIpAddress("130.242.197.92")
+        .withMethod(AuthMethod.QrCode)
+        .build();
+
+    bankid
```

Comment:
Preferably example code should use `async`/`await` instead of plain promises.

Reply:
I will change it.